caikit/interfaces/runtime/data_model/package.py

```diff
@@ -19,3 +19,4 @@
 from caikit.core.data_model import CAIKIT_DATA_MODEL
 
 RUNTIME_PACKAGE = f"{CAIKIT_DATA_MODEL}.runtime"
+RUNTIME_JOB_PACKAGE = f"{RUNTIME_PACKAGE}.job"
```

Comment:
This was required due to a [py-to-proto](https://github.com/IBM/py-to-proto/issues/66) bug with protobuf 3

Reply:
Yuck! I wonder if we should just fix that and bump the `py-to-proto` dependency here? That might be less painful than the inevitable breaking change when we want to consolidate the package naming.

Reply:
So I initially looked to fix `py-to-proto` but I had no idea where it lived 😅 . I'll look into that once I resolve the other comments

Reply:
I was able to resolve it! Its actually a bug in the underlying protobuf code so I implemented a fix in caikit